src/nvim/ex_cmds.c

```diff
@@ -4822,8 +4822,8 @@ static void helptags_one(char_u *dir, char_u *ext, char_u *tagfname,
 
   // Find all *.txt files.
   size_t dirlen = STRLEN(dir);
-  STRCPY(NameBuff, dir);
-  STRCAT(NameBuff, "/**/*");
+  STRLCPY(NameBuff, dir, sizeof(NameBuff));
+  STRCAT(NameBuff, "/**/*");  // NOLINT
```

Comment:
Uh, thanks for looking into it. I would have assumed that the `+` doesn't matter since it basically means to fetch more characters at once so that the outer regexp doesn't have to be repeated as often. Regexp suck!